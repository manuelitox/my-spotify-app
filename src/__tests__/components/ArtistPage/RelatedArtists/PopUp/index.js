import React from 'react'
import { shallow } from 'enzyme'

import PopUpRelatedArtists from 'components/ArtistPage/RelatedArtists/PopUp/'

describe('Component | PopUpRelatedArtists', () => {
  const relatedArtists = [
    {
      id: '123',
      name: 'artist name',
      images: [{ url: 'path/to/photo', height: 200 }]
    }
  ]
  let component 
  beforeEach( () => {
    component = shallow(
      <PopUpRelatedArtists isOpen togglePopUp={ jest.fn() } token="123abc" relatedArtists={ relatedArtists } />
    )
  })
  it('should render a component', () => {
    expect(component).toMatchSnapshot()
  })
})