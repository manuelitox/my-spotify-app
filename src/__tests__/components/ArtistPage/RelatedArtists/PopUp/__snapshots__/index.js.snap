// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component | PopUpRelatedArtists should render a component 1`] = `
<OnClickOutside(PopUpContainer)
  eventTypes={
    Array [
      "mousedown",
      "touchstart",
    ]
  }
  excludeScrollbar={false}
  isOpen={true}
  outsideClickIgnoreClass="ignore-react-onclickoutside"
  preventDefault={false}
  stopPropagation={false}
  togglePopUp={[MockFunction]}
>
  <Title
    color="#262D45"
    insidePopUp={true}
  >
    Related Artists
  </Title>
  <styled.div
    inPopup={true}
    onClick={[MockFunction]}
  >
    <Connect(Artist)
      id="123"
      key="0"
      name="artist name"
      photo="path/to/photo"
    />
  </styled.div>
</OnClickOutside(PopUpContainer)>
`;
